lms/djangoapps/course_api/blocks/views.py

```diff
@@ -128,6 +128,10 @@ class BlocksView(DeveloperErrorViewMixin, ListAPIView):
             the child blocks.  Returned only if "children" is included in the
             "requested_fields" parameter.
 
+          * completion: (float or None) The level of completion of the block.
+            Its value can vary between 0.0 and 1.0 or be equal to None
+            if block is not completable.
+
```

Comment:
This should mention that the value is only included if "completion" included in requested_fields.  See `graded` or `format` below.

Reply:
👍